src/libraries/System.Private.CoreLib/gen/IntrinsicsInSystemPrivateCoreLibAnalyzer.cs

```diff
@@ -0,0 +1,533 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Linq;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Operations;
+
+// This isn't a shipping analyzer, so we don't need release tracking
+#pragma warning disable RS2008
```

Comment:
I believe we disable Roslyn-style release tracking in this repo, so we can remove this suppression.

```suggestion
```

Reply:
Nope. It fails the build without disabling this warning.